.dockerignore

```diff
@@ -26,3 +26,4 @@ endo-sha.txt
 # `@@AGORIC_DOCKER_SUBMODULES@@`
 packages/xsnap/moddable
 packages/xsnap/xsnap-native
+!packages/xsnap/xsnap-native/xsnap/build/bin
```

Comment:
@turadg, I got a tip from @michaelfig that you might have pending changes to Docker stuff that might overlap with this effort to speed up `xsnap` builds. Please take a look and let me know whether the wind is carrying us in the same direction.

Reply:
My thing landed in a couple days ago: https://github.com/Agoric/agoric-sdk/pull/8850 Sorry I didn't think to tag you. I had picked the reviewers before you took up xsnap.

Your branch is on top of that from master so no potential for conflicts.

That change was to have a Docker layer specifically for xsnap, so that the `yarn install` didn't try to build it again if it hadn't changed. I think that's good to keep and should reduce the number of cases you have to consider.

Reply:
The Docker build of `xsnap` in a container with the build toolchain is great and I think does simplify the constraints on `build.js`. I’ve mostly adjusted the comments to reflect that simplification.